cloud_dashboard/src/container/LaunchTemplateView.tsx

```diff
@@ -0,0 +1,113 @@
+import DataTable from 'component/DataTable';
+import { AWS_LAUNCH_TEMPLATE_LIST, K8S_LAUNCH_TEMPLATE_LIST } from 'constant';
+import CloudContext from 'model/CloudContext';
+import DataColumn from 'model/DataColumn';
+import DataRecord from 'model/DataRecord';
+import EntityColumn from 'model/EntityColumn';
+import EntityData from 'model/EntityData';
+import SortInfo from 'model/SortInfo';
+import React, { useEffect, useState } from 'react';
+import HttpService from 'service/http';
+import { convertEntityData, readDataCache } from 'service/utility';
+
+/**
+ * Get LaunchTemplateColumnList by cloud_context.
+ * 
+ * @param cloudContext cloud_context.
+ * @returns LaunchTemplateColumnList.
+ */
+const getLaunchTemplateColumnList = (cloudContext: CloudContext): EntityColumn[] => {
+  switch (cloudContext.cloudServiceProvider) {
+    case 'aws_cloud':
+      if (cloudContext.name !== 'ALL') {
+        return AWS_LAUNCH_TEMPLATE_LIST;
+      } else {
+        return [
+          { labelName: 'Cloud Service Provider ID', name: 'cloud_context', type: 'default' },
+          ...AWS_LAUNCH_TEMPLATE_LIST
+        ];
+      }
+    case 'k8s':
+      if (cloudContext.name !== 'ALL') {
+        return K8S_LAUNCH_TEMPLATE_LIST;
+      } else {
+        return [
+          { labelName: 'Cloud Service Provider ID', name: 'cloud_context', type: 'default' },
+          ...K8S_LAUNCH_TEMPLATE_LIST
+        ];
+      }
```

Comment:
@RyoYamashita765 Can we refactor:

```suggestion
      return cloudContext.name !== 'ALL'
        ? K8S_LAUNCH_TEMPLATE_LIST
        : { labelName: 'Cloud Service Provider ID', name: 'cloud_context', type: 'default' },
          ...K8S_LAUNCH_TEMPLATE_LIST
          ];
```